src/unit_tests/app_test.py

```diff
@@ -92,8 +92,9 @@ def test_get_dataset_metadata(client, database):
     Checks that the API endpoint '/v1/dataset_metadata' returns the expected dataset dictionary object
     when invoked with the survey_id and period_id parameters.
     """
-    # Added "filename" field in the expected metadata
+    # Added "filename", "dataset_id" fields in the expected metadata
```

Comment:
Again I think the expected_metadata object is clear enough in what it does, I'd remove this comment